From ff8562f7724c4db4b83635af9e627f3495222327 Mon Sep 17 00:00:00 2001
From: Anibal Limon <limon.anibal@gmail.com>
Date: Tue, 22 Aug 2017 04:41:31 -0500
Subject: [PATCH 1/3] apt-pkg/contrib/strutl.cc: Include array header

If GCC version is less than 5 the array header needs to be included
to support std::array.

After we drop debian8 and centos7 that has gcc < 5 versions this patch
can be remove.

Signed-off-by: Anibal Limon <limon.anibal@gmail.com>

Upstream-Status: Inappropriate [embedded specific]
---
 apt-pkg/contrib/strutl.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/apt-pkg/contrib/strutl.cc b/apt-pkg/contrib/strutl.cc
index 60d0ca8..c2ff01d 100644
--- a/apt-pkg/contrib/strutl.cc
+++ b/apt-pkg/contrib/strutl.cc
@@ -27,6 +27,7 @@
 #include <sstream>
 #include <string>
 #include <vector>
+#include <array>
 
 #include <stddef.h>
 #include <stdlib.h>
-- 
2.1.4

